recipes/makim/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "makim" %}
+{% set version = "1.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/makim-{{ version }}.tar.gz
+  sha256: d7db0a978aa634d40896043057df11ea908d982080365e409b173c3e35116369
+
+build:
+  skip: True  # [win]
```

Comment:
This can probabley be noarch. `sh` should work on Windows bash, right?

Reply:
mm not sure .. checking sh feedstock I saw this: https://github.com/conda-forge/sh-feedstock/blob/main/recipe/meta.yaml#L24

maybe we could remove this skip and add `__unix` similar to what `sh` is using.
maybe this is what you @ocefpaf meant?

@felipepaes could you try that please?

PS: this noarch and skip thing always get me confused XD

Reply:
Yep. You can make it noarch in the same fashion as `sh`. However, I'd love to test `sh` with a Windows shell, but that is an exercise for the future 😄

Reply:
ohhh got it .. I didn't know that .. this is really new to me .. sorry for the confusion XD
we can try that as well .. if you have any example about that it we could try for sure

Reply:
You can use the `sh` package as an example.

Reply:
awesome! ok thank you so much!